src/legacy/ui/public/vislib/__tests__/lib/fixtures/dispatch_heatmap_data_point.json

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
Nit: Can we strip down these fixtures somehow in a reasonable way? They are really large which makes them hard to grasp for a reader.

Reply:
They were originally much larger :) 10000 lines+ 
I'll do my best!

Reply:
I did my best, now I can call myself Zip